library/core/src/marker.rs

```diff
@@ -469,6 +469,12 @@ pub macro Copy($item:item) {
 #[cfg_attr(not(test), rustc_diagnostic_item = "Sync")]
 #[lang = "sync"]
 #[rustc_on_unimplemented(
+    on(_Self = "std::cell::OnceCell<T>", note = "use `std::sync::OnceLock` instead"),
+    on(
+        _Self = "std::cell::Cell<T>",
+        note = "use `std::sync::RwLock` or an atomic variable instead",
```

Comment:
> or an atomic variable instead

Do you mean an atomic integer? That'll only map with `Cell<{integer}>`  where integer != i128/u128

Reply:
Can we do
```
on(_Self = "std::cell::Cell<bool>", note = "...AtomicBool..."),
on(_Self = "std::cell::Cell<u8>", note = "...AtomicU8..."),
...
```
?